.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Checks:
+
+* [ ] Update changelog
+* [ ] Rebase to `dev` branch
+* [ ] Assign reviewers
+
```

Comment:
Should we add [ ] `Assign project` as we're using `Zenhub` to track issues?

Reply:
There's no need for that. See https://github.com/mapbox/mapbox-vision/issues/951